docs/operations.md

```diff
@@ -128,6 +128,9 @@ times to job events using various filters in order to create a more advanced eve
 
 ```bash
 wfxctl job events
+
+# auto reconnect if connection is lost; also waits for wfx to be up and running
+wfxctl job events --auto-reconnect
```

Comment:
Here's probably a good place to detail the implications of this flag (e.g., possibly lost events from a client's view).

Reply:
Done